Agents/CEAAgent/README.md

```diff
@@ -69,59 +69,48 @@ The CEA Agent is reachable at http://localhost:3838/cea-agent, where 3838 is the
-
-Example request:
-```
-{ "iris" :
-["http://www.theworldavatar.com:83/citieskg/namespace/kingslynnEPSG27700/sparql/cityobject/UUID_0595923a-3a83-4097-b39b-518fd23184cc/"],
-"geometryEndpoint" : "http://host.docker.internal:48888/kingslynnEPSG27700",
-"ceaEndpoint": "http://host.docker.internal:48888/cea",
-"graphName": "http://127.0.0.1:9999/blazegraph/namespace/cea/cea"}
+"geometryEndpoint" : "http://localhost:3838/access-agent/kingslynnEPSG27700",
+"ceaEndpoint": "http://localhost:3838/access-agent/outputs"}
 ```
-In the above request example, the CEA Agent will be querying building geometry from the Blazegraph that ```http://host.docker.internal:48888/kingslynnEPSG27700``` is pointed to. All the other inputs for CEA will be queried from the default endpoints within stack as specified in ```./cea-agent/src/main/resources/CEAAgentConfig.properties```. The CEA triples will be instantiated under the ```http://127.0.0.1:9999/blazegraph/namespace/cea/cea``` graph in the Blazegraph where ```http://host.docker.internal:48888/cea``` is pointed to.
+In the above request example, the CEA Agent will be querying building geometry from the Blazegraph that ```http://localhost:3838/access-agent/kingslynnEPSG27700``` is pointed to. All the other inputs for CEA will be queried from the default endpoints within stack as specified in [CEAAgentConfig.properties]. The CEA triples will be instantiated in the Blazegraph where ```http://localhost:3838/access-agent/outputs``` is pointed to.
 
 Example request:
 ```
 { "iris" :
 ["http://www.theworldavatar.com:83/citieskg/namespace/kingslynnEPSG27700/sparql/cityobject/UUID_0595923a-3a83-4097-b39b-518fd23184cc/"]}
 ```
-In the above request example, the CEA Agent will be querying building geometry from the ```citieskg-kingslynnEPSG27700``` namespace in TheWorldAvatar Blazegraph.  All the other inputs for CEA will be queried from the default endpoints within stack as specified in ```./cea-agent/src/main/resources/CEAAgentConfig.properties```. The CEA triples will be instantiated in the stack Blazegraph.
+In the above request example, the CEA agent will be all the CEA inputs from the default endpoints within stack as specified in [CEAAgentConfig.properties]. The CEA triples will be instantiated in the stack Blazegraph.
 
 ### 3.2. Update
 
 Available at http://localhost:3838/cea-agent/update.
 
-Requests to the update endpoint is automatically sent by the CEA Agent after running and receiving requests to the run endpoint. The update endpoint updates the knowledge graph with CEA outputs.
+Requests to the update endpoint is automatically sent by the CEA agent after running and receiving requests to the run endpoint. The update endpoint updates the knowledge graph with CEA outputs.
```

Comment:
Does this mean that a 'user' should never call this route (at least under normal circumstances)? If so, please state that here explicitly.

Reply:
User should not call this route. Added clarification.